.ci/jobs.t/elastic+elasticsearch+periodic+dra-snapshots-trigger.yml

```diff
@@ -0,0 +1,6 @@
+---
+jjbb-template: periodic-trigger-lgc.yml
```

Comment:
We can start with this, but I think we'll want to publish snapshots on every successful intake commit. I don't see any reason not to.

Reply:
I'm happy with every successful commit. I just used that as it is DAYLY releasable artifacts. but maybe took it to literately :)

Reply:
_At least_ daily 😉